ubyssey/static/src/styles/modules/widgets/_upcoming_events_horizontal.scss

```diff
@@ -0,0 +1,51 @@
+.c-widget-upcoming-events-horizontal {
+  background-color: #f5f5f5;
+  width: 100%;
+  margin: 15px 0;
+
+  box-sizing: border-box;
+  padding: 25px 50px;
+  border-top: solid 1px $color-border-gray;
+  border-bottom: solid 1px $color-border-gray;
+}
+
+.c-widget-upcoming-events-horizontal__label-row {
+  width: 100%;
+  overflow: hidden;
+
+  // Text
+  color: $color-text-gray;
+  text-align: center;
+  font-size: font-size(14);
+  font-weight: $font-weight-semi-bold;
+  letter-spacing: 0.045em;
```

Comment:
Add this style and then make the actual HTML lowercase (better for web crawlers):

```scss
text-transform: uppercase;
```